xbmc/network/GUIDialogNetworkSetup.h

```diff
@@ -24,6 +24,7 @@ class CGUIDialogNetworkSetup : public CGUIDialogSettingsManualBase
     int defaultPort;       //!< Default port to use for protocol
     std::string type;      //!< URL type for protocol
     int label;             //!< String ID to use as label in dialog
+    std::string addonId; //!< Addon identifier, leaved empty if inside Kodi
```

Comment:
It's not evident to me, where is this used?

Reply:
Currently vfs addons are forced to take Kodi's own string id's (e.g. at `vfs.sftp` https://github.com/xbmc/vfs.sftp/blob/Leia/vfs.sftp/addon.xml.in#L21) and the `std::pair<int, int>` in settings this could not be identified.

This is then taken here https://github.com/xbmc/xbmc/pull/17527/files#diff-8211fa801621c92df9bf6515d2d45b8aR200 to give to replaced `pair` with `TranslatableIntegerSettingOption` his addon id.

Reply:
thanks for the explanation